test/tests/functional/pbs_cray_check_node_exclusivity.py

```diff
@@ -86,7 +87,7 @@ def test_node_state_with_adavance_resv(self):
         self.server.expect(NODE, {'state': 'free'},
                            id=resv_node)
         self.server.qterm(runas=ROOT_USER)
-        self.server.start()
+        self.server.restart()
```

Comment:
You don't need to qterm() and then restart().  restart() will stop the server first.

Reply:
Done.